docs/installation.rst

```diff
@@ -0,0 +1,17 @@
+.. _installation:
+
+Installation
+============
+
+RabbitMQ
+--------
+
+Several of nameko's built-in features rely on RabbitMQ.
```

Comment:
+1 for link to rabbit docs, but can also include basics on ox x/ubuntu (brew install / apt-get install)